patches/buildroot/0005-Bring-back-sysroot-fix-for-binutils.patch

```diff
@@ -0,0 +1,195 @@
+From 9afc3195d55b209fbf6668ed159198a930b9163b Mon Sep 17 00:00:00 2001
+From: Martin Harriman <larvacea@mac.com>
+Date: Mon, 18 Nov 2024 23:00:05 +0000
+Subject: [PATCH 5/6] Bring back sysroot fix for binutils
```

Comment:
Did they mention why the removed these patches?

Reply:
I do not know. This patch-within-a-patch is present in the current SDK. It had to change to this form because it applies to a versioned subdirectory in buildroot. The current patch applied to the 2.38 subdirectory, no longer present as of buildroot 2023.08.4, and needs now to apply to the versioned subdirectories in buildroot 2023.08.4.